# HG changeset patch
# User Philip Nienhuis <prnienhuis@users.sf.net>
# Date 1355433276 -3600
#      Thu Dec 13 22:14:36 2012 +0100
# Node ID 2abea2cfdace38e926110674b5944a08999ec6f3
# Parent  d61b92690820d90c27b331f86e31f0b1106455d0
java_new.m: can also have only one argument

diff --git a/scripts/deprecated/java_new.m b/scripts/deprecated/java_new.m
--- a/scripts/deprecated/java_new.m
+++ b/scripts/deprecated/java_new.m
@@ -36,16 +36,16 @@ function retval = java_new (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
              "java_new is obsolete and will be removed from a future version of Octave; please use javaObject instead");
   endif
 
-  if (nargin < 2)
+  if (nargin < 1)
     print_usage ();
   endif
 
   retval = javaObject (varargin);
 
 endfunction
 
